src/js/steps/letters/generateHtml.js

```diff
@@ -94,7 +94,7 @@ module.exports = function (options) {
         // If we're showing the duplicate letter overlay, and this is one of the
         // duplicate letters, we add a class to show visually that this is a
         // changed letter.
-        if (letter.selected !== letter.default_character && options.showOverlay) {
+        if (letter.selected !== letter.default_character && options.showOverlay && determineIfDuplicate()) {
```

Comment:
Can you stick this entire line in a `hasChanged` variable? https://github.com/lostmyname/js-style-guide#use-descriptive-conditions